mne/time_frequency/multitaper.py

```diff
@@ -543,12 +540,68 @@ def multitaper_psd(x, sfreq=2 * np.pi, fmin=0, fmax=np.inf, bandwidth=None,
                        for x in np.array_split(x_mt, n_jobs))
         psd = np.concatenate(out)
 
-    if x.ndim == 1:
-        # return a 1d array if input was 1d
-        psd = psd[0, :]
-
     freqs = freqs[freq_mask]
     if normalization == 'full':
         psd /= sfreq
 
+    # Combining/reshaping to original data shape
+    psd = psd.reshape(np.hstack([dshape, -1]))
+    if ndim_in == 1:
+        psd = psd.squeeze()
```

Comment:
squeeze is evil. It squeezes all the dimensions equal to 1. I've been bitten by this in the past. If you know it's the first dim that is one just do as it was before `psd = psd[0, :]`